src/components/VContentLink/VContentLink.vue

```diff
@@ -36,6 +40,10 @@ export default defineComponent({
       type: Number,
       required: true,
     },
+    isSelected: {
+      type: Boolean,
+      default: false,
+    },
   },
   setup(props) {
     const { i18n } = useContext()
```

Comment:
We should probably extract this `resultsCountLabel` function in the future, because it's used in the search bar as well.

Reply:
I extracted it into a new `use-i18n-utilities.js` composable and tried to replace the code on the VHeader but seems not to work well within `watchEffect`, I couldn't make it work there.